Google.Api.Generator.Tests/ProtoTests/CommonResource/CommonResource.proto

```diff
@@ -15,15 +15,11 @@ service CommonResource {
   }
 }
 
-// This is defined as a common resource for C#, so will not have a class generated for it.
-// See CommonResourceConfig.json
-message Project {
-  option (google.api.resource) = {
+// Also defined as a common resource in CommonResourceConfig.json
+option (google.api.resource_definition) = {
     type: "commonresource.example.com/Project",
 	pattern: "projects/{project}"
-  };
-  string name = 1;
-}
+};
```

Comment:
Is there a reason why pattern is indented further than type here?

Reply:
Tab/space confusion. Fixed.